test/e2e/specs/specs-playwright/wp-editor__post-basic-flow-spec.ts

```diff
@@ -1,5 +1,6 @@
 /**
  * @group calypso-pr
```

Comment:
As an aside, I think Gutenberg editor flows should not be part of the `calypso-pr` group.

Reply:
Leaving this one for now - we can revisit later!